src/components/APITokenDialog/index.tsx

```diff
@@ -166,7 +166,7 @@ const APITokenDialog: FC<Props> = ({ onClose, open, ...rest }) => {
   };
 
   const generateToken = async () => {
-    if (!hasPermission(user, "data_submission", "create")) {
+    if (!hasPermission(user, "data_submission", "create", null, true)) {
```

Comment:
I know this permission/role check predates these changes, but it's worth questioning if we still need this here? Since both the parent component (navbar in this case) and the API also check PBAC, is this redundant? 

Just a thought.

Reply:
Agreed, we can assume we already checked for the permission from the navbar. Check removed in [3a65096](https://github.com/CBIIT/crdc-datahub-ui/pull/573/commits/3a6509668419403e59eda0dff80d9d86367f9d79).